src/_utils/offline/service-worker/sw.template.js

```diff
@@ -0,0 +1,57 @@
+const cacheName = "<%= cacheName %>"
+
+/* eslint-disable quotes */
+const cacheList = JSON.parse('<%= files %>')
```

Comment:
does this list will contains all files? Like those already defined in `window.__COLLECTION__`?